pkgs/development/tools/hotdoc/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, python3Packages
+, pkg-config
+, cmake
+, glib
+, json-glib
+, libxml2
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "hotdoc";
+  version = "0.12.2";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0nypvcqp4b0qfps35f7mz3d90zjh0v5wawpd06ra28q92nxblx17";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
```

Comment:
Do not we need `flex`: https://github.com/hotdoc/hotdoc/blob/83d0c70ecb9e0224ec67dcf7c6c5cfee8d049288/setup.py#L389-L422

Reply:
:+1:.